### ISSUE TITLE
🚜 Event bulk moving and copying

### ISSUE THREAD

**Issue by richardolsson** [zetkin/app.zetkin.org#1251]
## Description
A user should be able to move or copy events in bulk.

## Relevant Job Stories
* When I’m organizing a large campaign with small events that are mostly repetitive “actions”, I want to reuse information between events, so that I don’t have to fill out the same information over and over again

## Prerequisites
Don't start this until #1249 has been completed.

## Requirements
- [x] In the selection mode panel, display buttons called "Copy" and "Move"
- [x] Clicking either should open a menu with different options for how to copy and move respectively
- [ ] Picking an option should move/copy as such:
  - [x] Move to "Next week" should postpone all selected events seven days.
  - [x] Move to "Next day" should postpone all selected events one day.
  - [ ] Copy to "Next week" should create copies where start and end dates are seven days in the future relative to the originals.
  - [ ] Copy to "Next day" should create copies where start and end dates are one day in the future relative to the originals.
  - [x] Copy to "Same date & time (duplicate)" should create exact copies of all selected events
  - [x] Copy to "After each event (create shifts)" should copy all selected events and move the new ones so that they happen right after originals, i.e. their start dates become the end date of the original, and the end date is updated to retain the original duration.
- [x] Move or Copy to "Next week" should display the date events will span if postponed seven days.
- [x] Move or Copy to "Next day" should display the dates events will span if postponed one day
## Possible implementations
None

### Selecting
![image](https://user-images.githubusercontent.com/550212/233641593-e736102e-bd2b-446b-91af-5f3308276fe2.png)

### Move options
![image](https://user-images.githubusercontent.com/550212/233642805-3a1baf9f-dd2f-4be8-94d9-cabc3dcae04f.png)

### Copy options
![image](https://user-images.githubusercontent.com/550212/233642773-22e3a012-4ce7-4fdb-b3ac-6ae0b6bdd0f4.png)

## Open questions
None
